blacklight-cornell/config/routes.rb

```diff
@@ -47,6 +47,7 @@
 
     collection do
       delete "clear"
+      post "track" # Define the track route locally so the `track_bookmarks_path` helper remains available after code reloads.
```

Comment:
This was the fix that I came up with that saved me some headache from having to constantly reload the rails app after every comment, or code change.